clients/python/llmengine/model.py

```diff
@@ -68,6 +72,13 @@ def create(
             num_shards (`int`):
                 Number of shards for the LLM. When bigger than 1, LLM will be sharded
                 to multiple GPUs. Number of GPUs must be larger than num_shards.
+                Only affects behavior for text-generation-inference models
+
+            quantize (`Optional[Quantization]`):
+                Quantization for the LLM. Only affects behavior for text-generation-inference models
+
+            checkpoint_path (`Optional[str]`):
```

Comment:
Can I get a bit more info here (or link to info) on the requirements for the checkpoint? i.e does it need to be a .tar file? do weights need to be saved as safetensors?